### ISSUE TITLE
cherry pick rebase first aeon commit.

### ISSUE THREAD

**Issue by guzzijones** [drparadoxical/aeon-rebase#8]
I suspect this one will be difficult.  This would be the first aeon commit not shared with monero. More than likely it contains a lot of fluf like licensing etc.

**Comment by drparadoxical** [comment]
I was thinking similarly, except that fluff would include usage. One of the things the community is often asking, is how to use any given thing, and as a first commit not shared with Monero, our first commit(s) be to update the README.md and that it include an exhaustive breakdown of usage per executable. 

I've got that exhaustive breakdown already for Aeon v0.9.14.0, and am updating it relative to the additional flags, commands, calls made available with this rebase for the daemon, simplewallet, etc. 

@guzzijones, you mentioned issue #9 was something you wanted to do first, I'll reply to you about that, there, in a minute. 

Let me know what you think of my suggestion about for the first commit, but no one need wait to just take the initiative and push in whatever, whenever, accordingly. As I said before, there's no need to wait on approval or for documentation to be pushed into aeon-project-planning, as I'm focused on keeping things tight and together, not on telling you what to do.

**Comment by guzzijones** [comment]
I was referring more to the first aeon commit from the aeon branch. But you bring up a bigger issue. Yes we need to clean the readme. Do we do that now or after commits from aeon. Because likely they cross over.

**Comment by guzzijones** [comment]
All in all it doesn't matter either way. So by all means update the readme with whatever you want. We have that first commit cleaned up so we are ready for takeoff.

**Comment by drparadoxical** [comment]
Been a long day, apologies for being dense, just not fully understanding what you mean. Let me tell you how I see it and if I'm still not understanding you, please keep trying. 

Right now, having forked not from aeonix/aeon, but aeonix/aeon-rebase, our starting point was the monero-project/monero tag, so the issues that exist for aeonix/aeon are, by and large, non-issues now. This, and again from how I'm looking at it all and from my perspective, means that we neither have to address the open issues for aeonix/aeon, nor those for monero-project/monero, but work against this fork here in this drparadoxical/aeon-rebase repo. 

We also don't have to fully bring this up to the monero-project/monero tag functionality, either, because it appears the stopping point, defined by Smooth in the guidelines Reddit post, is just before RCT related work, so our focus here is going to be on the work between where we are right now, and that moment before RCT is integrated and becomes relevant. 

Again, if I'm the one not getting it, keep beating me with that stick you've got there, want to make sure I'm understanding you properly.

**Comment by guzzijones** [comment]
This issue needs to wait until we have coveralls and buildbot up.  So I am getting ahead of ourselves a bit.

So there should be commits in aeon and monero with the same hash.  Those are from bytecoin.  After that Aeon should have a commit that this "monero" repo (we are using to rebase against) does not.  Basically we don't need to reinvent the wheel here, but just merge that commit and then keep merging through the aeon commits until we have all of them. 

None of the AEON commits should have ringct.  So that shouldn't be an issue.

ELI5:
In essence reapplying the first AEON ONLY commit to this "Monero" base.
This PR is for someone to find that first one.  and try it out. :+1:

**Comment by drparadoxical** [comment]
I don't see it as getting ahead of yourself, I think it's simply an aspect of the CI flow of which I was wholly unaware. Whenever work is done, it's committed, that submission should be assumed to have been tested and then tested again, and finally, once the submission has been verified as correct and understood to be complete relative to the task or issue it was meant to address, the work is merged into the trunk accordingly in preparation for tag, pre-production, phase shift, and version support and maintenance. Business as usual. 

All you're saying, if I'm hearing you correctly, is that we're going to feed AEON commits sequentially and systematically through that cycle, so these systems do needs to be set up now, and we can get the kinks worked out as they occur per AEON commit. 

I'm assuming since you said coveralls, that a previous comment I made and an assumption I stated was, in fact, correct, in that we need a VPS or VM with Buildbot and Coveralls set up, and unless I'm told otherwise, Coverity as a part of the CI flow as well.

**Comment by guzzijones** [comment]
Yes exactly.  This issue can be closed moved to Milestone 2.  Milestone 1 is basically: "get shit setup. " :+1:

**Comment by drparadoxical** [comment]
Indeed.

**Comment by drparadoxical** [comment]
This issue was moved to drparadoxical/aeon-rebase-1#3
